src/app/core/components/add-marios-form/components/title-text-area/title-text-area.component.scss

```diff
@@ -0,0 +1,15 @@
+@import "../../../../../shared/variables/global.scss";
+
+textarea {
+  border: 1px solid $color-text-gray-light;
+  border-radius: $border-radius;
+  resize: none;
+  width: 100%;
+  height: 100px;
```

Comment:
To consider: Maybe rem? Or you want to have 100px on all devices and even if page will have different zoom value?